app/views/admin/facility_groups/_form.html.erb

```diff
@@ -25,7 +24,7 @@
           organizations.pluck(:name, :id),
           {},
           {disabled: action_name == "edit"}) %>
-      <%= form.text_field :name, id: :facility_name, required: true, help: "Usually the name of a District", autocomplete: "off" %>
+      <%= form.text_field :name, id: :facility_name, required: true, autocomplete: "off" %>
```

Comment:
Is the ID used? If not, we can axe/fix it. It should be `facility_group_name`, yes?

Reply:
Leaving as-is so we don't accidentally break anything.